website/identifiers/clients/datacite.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import re
+import datetime
+
+from website.util.client import BaseClient
+from website import settings
+from datacite import DataCiteMDSClient, schema40
+
+
+class DataCiteClient(BaseClient):
```

Comment:
I'm not sure it makes sense to inherit from `BaseClient`, since you're not using any of its methods.

Also, it's not immediately obvious what methods MUST be implemented by an identifier client. From comparing `datacite.py` and `crossref.py`, it looks like `create_identifier` and `change_status_identifier` are required, but one can't be certain without doing a code dive.

How about defining either an interface (e.g. `AbstractIndentifierClient`) or a base class (e.g. `BaseIdentifierClient`)? Use an interface if you only want to enforce/document required methods. Use a base class if there will be shared behavior among subclasses.